packages/myst-cli/src/transforms/code.spec.ts

```diff
@@ -114,3 +114,65 @@ describe('liftCodeMetadataToBlock', () => {
     expect(mdast.children[0].children[1].value).toEqual('print("hello world2")');
   });
 });
+
+function build_mdast_by_tags(tags: string[]) {
+  const mdast: any = {
+    type: 'root',
+    children: [
+      {
+        type: 'block',
+        children: [
+          {
+            type: 'code',
```

Comment:
I think adding an `executable: true` here is reasonable!

perhaps extending this test to `not` have an output as well, which I think would break things at the moment!